src/lib/game/world/handler.js

```diff
@@ -112,16 +114,7 @@ class WorldHandler extends EventEmitter {
     });
   }
 
-  changeModel(_unit, oldModel, newModel) {
-    // Only need to handle skeleton helper changes here
-    if (oldModel && oldModel.skeletonHelper) {
-      this.scene.remove(oldModel.skeletonHelper);
-    }
-
-    if (newModel) {
-      newModel.skeletonHelper = new THREE.SkeletonHelper(newModel);
-      this.scene.add(newModel.skeletonHelper);
-    }
+  changeModel(_unit, _oldModel, _newModel) {
```

Comment:
So I'd love to get the ```SkeletonHelper``` working again, but since I disabled matrixAutoUpdate in most places, it doesn't like to translate with the M2 anymore.

You should still be able to do ```displayID = 12831``` in ```lib/game/player.js```, though.